azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -131,7 +131,8 @@ internal class ParticipantListView(
                 localParticipantCellModel.displayName, localParticipantCellModel.isMuted
             )
         )
-        for (remoteParticipant in remoteParticipantCellModels) {
+        val sortedRemoteParticipantCellModels = remoteParticipantCellModels.sortedBy { it.displayName }
```

Comment:
This sort is case sensitive. I would prefer to ignore case for this.
You can also confirm with Alex his preference.

Reply:
after confirm with Alex, we will sort list without case sensitive. code updated.